src/main/java/seedu/address/ui/UiPart.java

```diff
@@ -15,7 +15,7 @@
 public abstract class UiPart<T> {
```

Comment:
actually we not using this later right

Reply:
you are right. My import statements had issues